src/main/java/org/isf/medicalstock/manager/MovStockInsertingManager.java

```diff
@@ -58,8 +59,10 @@ public MovStockInsertingManager(MedicalStockIoOperations medicalStockIoOperation
 	 * 
 	 * TODO: verify the need of checkReference param
 	 *
-	 * @param movement - the movement to validate
-	 * @param checkReference - if {@code true} it will use {@link #checkReferenceNumber(String) checkReferenceNumber}
+	 * @param movement       - the movement to validate
+	 * @param checkReference - if {@code true} it will use
```

Comment:
So many of the changes are just formatting; and it looks like you aren't using the project default style.